src/Microsoft.DotNet.Interactive.HttpRequestParser/HttpRequestNode.cs

```diff
@@ -70,10 +74,9 @@ internal class HttpRequestNode : HttpSyntaxNode
     public HttpBindingResult<HttpRequestMessage> TryGetHttpRequestMessage(HttpBindingDelegate bind)
     {
         var request = new HttpRequestMessage();
-        var diagnostics = new List<Diagnostic>();
-        var success = true;
+        var diagnostics = new List<Diagnostic>(base.GetDiagnostics());
 
-        if (MethodNode is { Span.IsEmpty: false })
+        if (MethodNode is { FullSpan.IsEmpty: false })
```

Comment:
minor: Shouldn't this still be `.Span`? This condition will currently return `true` if the node happens to contain only whitespace. Is the `HttpMethod` constructor tolerant of empty string it will be passed to it in such cases?